src/Plugin/SecurityChecker.php

```diff
@@ -91,6 +91,8 @@ public function execute()
             if ($this->allowedWarnings != -1 && ($result->count() > $this->allowedWarnings)) {
                 $success = false;
             }
+        } elseif (null === $warnings && $result) {
+            $this->builder->logWarning('invalid json: '.$result);
```

Comment:
maybe throwing a RuntimeException (or another Exception) would be better.

Reply:
Yes, throwing a RuntimeException would be great.